app/_kong_entities/service.md

```diff
@@ -5,14 +5,16 @@ entities:
 
 content_type: reference
 
+description: In Kong Gateway, a service is an abstraction of an existing upstream application. Services can store collections of objects like plugin configurations, and policies, and they can be associated with routes.
```

Comment:
I like this a lot. We can then require page to have a description (maybe limit it to some character count)? Then we can pull it in any place that we do link previews, including SEO reasons - this is the description that should get pushed to Google.